applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, model, custom_settings):
             "block_builder": true,
             "clear_storage": false,
             "move_mesh_flag": true,
-            "multi_point_constraints_used": false,
+            "multi_point_constraints_used": true,
```

Comment:
@RiccardoRossi now the constraints B&S is enabled by default in StructuralMechanics, as requested